ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -360,15 +359,11 @@ - (dispatch_queue_t)methodQueue {
 }
 
 RCT_EXPORT_METHOD(setColorTheme:(IBGColorTheme)colorTheme) {
-    [[NSRunLoop mainRunLoop] performBlock:^{
```

Comment:
Why is this and `setPrimaryColor` removed with no replacement?

Reply:
It works without needing to run on the main loop, i tested several cases and it seems to always work as expected, so i removed it.